cmake/pkgconfig.in

```diff
@@ -8,4 +8,4 @@ Version: v@OSRM_VERSION_MAJOR@.@OSRM_VERSION_MINOR@.@OSRM_VERSION_PATCH@
 Requires:
 Libs: -L${libdir} -losrm
 Libs.private: @ENGINE_LIBRARY_LISTING@
-Cflags: -I${includedir} -I${includedir}/osrm
+Cflags: -I${includedir} -I${includedir}/osrm -I@CMAKE_INCLUDE_PATH@ @CMAKE_CXX_FLAGS@
```

Comment:
`@CMAKE_INCLUDE_PATH@` is a local include path and shouldn't go in here otherwise this seems fine.